internal/app/admin/http/cache_test.go

```diff
@@ -699,6 +505,42 @@ func TestAdminServer_CacheDumpHandler_WildcardSuffix_NotFound(t *testing.T) {
 	}
 }
 
+func verifyCacheOutput(t *testing.T, rr *httptest.ResponseRecorder, cdsFile string, ldsFile string) {
+	var actualResponse map[string]interface{}
+	err := json.Unmarshal(rr.Body.Bytes(), &actualResponse)
+	assert.NoError(t, err)
+
+	filecontentsCds, err := ioutil.ReadFile(cdsFile)
+	assert.NoError(t, err)
+	var expectedCdsResponse map[string]interface{}
+	err = json.Unmarshal(filecontentsCds, &expectedCdsResponse)
+	assert.NoError(t, err)
+
+	filecontentsLds, err := ioutil.ReadFile(ldsFile)
+	assert.NoError(t, err)
+	var expectedLdsResponse map[string]interface{}
+	err = json.Unmarshal(filecontentsLds, &expectedLdsResponse)
+	assert.NoError(t, err)
+
+	actualCacheResponse := actualResponse["Cache"].([]interface{})
+	assert.Equal(t, len(actualCacheResponse), 2)
+	var actualLdsResponse map[string]interface{}
+	var actualCdsResponse map[string]interface{}
+
+	if actualCacheResponse[0].(map[string]interface{})["Key"] == "test_lds" {
+		actualLdsResponse = actualCacheResponse[0].(map[string]interface{})
```

Comment:
for my education what is causing the `0` and `1` index difference?

Reply:
may be we use a for loop so that we arent dependent on exact indexes.

Reply:
The order of what gets printed out by the cache depends on the order of keys returned by `GetDownstreamAggregatedKeys()` when first searching through the cache. Since that function returns of map of keys, there is no guaranteed cache output order.

Reply:
Would this be functionally equivalent ?
```
for c := range actualCacheResponse {
  if c..(map[string]interface{})["Key"] == "test_lds" {
    actualLdsResponse = c.(map[string]interface{})
  } else {
    actualCdsResponse = c.(map[string]interface{})
  }
}
```

Reply:
Yes, I can incorporate that change.